packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md

```diff
@@ -85,13 +107,39 @@ As Flutter widgets are instrumented, you just need to turn on the leak tracking.
 
 Invoke `ensureInitialized` and `startAppLeakTracking` before `runApp` like [the example app does](https://github.com/polina-c/spikes/blob/master/leaking_app/lib/main.dart#L7) and then follow the steps for demo app.
 
-### Add instrumentation to your classes
+### Add instrumentation to your classes <a id='instrument'></a>
 
-Invoke `startObjectLeakTracking` in constructor or initializer (that is invoked only once), and `registerDisposal` in the method `dispose`
+The tool needs to know which objects to track and it needs to know when disposal for these objects happened.
+
+To provide this information to the tool, invoke `startObjectLeakTracking` in constructor or initializer (that is invoked only once), and `registerDisposal` in the method `dispose`
```

Comment:
"To provide this information to the tool for objects of a certain class invoke `startObjectLeakTracking` in the constructor..."

"... and invoke ..."